// runescape-api/index.js
"use strict";
var __createBinding = (this && this.__createBinding) || (Object.create ? (function(o, m, k, k2) {
    if (k2 === undefined) k2 = k;
    Object.defineProperty(o, k2, { enumerable: true, get: function() { return m[k]; } });
}) : (function(o, m, k, k2) {
    if (k2 === undefined) k2 = k;
    o[k2] = m[k];
}));
var __setModuleDefault = (this && this.__setModuleDefault) || (Object.create ? (function(o, v) {
    Object.defineProperty(o, "default", { enumerable: true, value: v });
}) : function(o, v) {
    o["default"] = v;
});
var __importStar = (this && this.__importStar) || function (mod) {
    if (mod && mod.__esModule) return mod;
    var result = {};
    if (mod != null) for (var k in mod) if (k !== "default" && Object.prototype.hasOwnProperty.call(mod, k)) __createBinding(result, mod, k);
    __setModuleDefault(result, mod);
    return result;
};
Object.defineProperty(exports, "__esModule", { value: true });
exports.runemetrics = exports.miscellaneous = exports.hiscores = exports.grandexchange = exports.clan = exports.bestiary = void 0;
const osrs = __importStar(require("./osrs"));
const bestiary = __importStar(require("./runescape/bestiary"));
exports.bestiary = bestiary;
const clan = __importStar(require("./runescape/clan"));
exports.clan = clan;
const grandexchange = __importStar(require("./runescape/grandexchange"));
exports.grandexchange = grandexchange;
const hiscores = __importStar(require("./runescape/hiscores"));
exports.hiscores = hiscores;
const miscellaneous = __importStar(require("./runescape/miscellaneous"));
exports.miscellaneous = miscellaneous;
const runemetrics = __importStar(require("./runescape/runemetrics"));
exports.runemetrics = runemetrics;
exports.default = {
    bestiary,
    clan,
    grandexchange,
    hiscores,
    miscellaneous,
    runemetrics,
    osrs,
};
